extensions/positron-r/package.json

```diff
@@ -194,6 +194,11 @@
             "type": "boolean",
             "default": false,
             "description": "%r.configuration.symbols.includeAssignmentsInBlocks.description%"
+          },
+          "positron.r.workspaceSymbols.includeCommentSections": {
```

Comment:
So all of these settings that we have here in this extension that include `positron` in their names need to be changed, since [_all_ settings are Positron settings](https://connect.posit.it/positron-wiki/extension-naming-guide.html). What will need to happen is that we:

- deprecate the original setting using `deprecationMessage`: https://code.visualstudio.com/api/references/contribution-points#Configuration-property-schema
- migrate the setting, similar to [what we've done here with "prereleases"](https://github.com/posit-dev/positron/pull/8610/files)

Can we avoid adding more to this future task? I would suggest `ark.workspaceSymbols.includeCommentSections` since all the smarts are coming from ark.

Reply:
Worth noting that these ark settings will probably have to migrate to a new namespace again within the next year when the LSP becomes independent of the kernel.

Reply:
We chatted elsewhere and cannot come up with a good alternative that will keep us from needing to migrate in the future. So we'll just take our lumps here, and sign up for _more_ future pain. 😭